draft-ietf-jose-hpke-encrypt.md

```diff
@@ -476,6 +489,20 @@ When using Key Encryption, the strength of the content encryption algorithm shou
 
 This document adds entries to {{JOSE-IANA}}.
 
+## Updates to "JSON Web Key Use" Registry
+
+The "JSON Web Key Use" registry is updated as follows:
+
+   o  Use Member Value: "HPKE-Sender"
+   o  Use Description: Key for HPKE sender role (encapsulation)
+   o  Change Controller: IESG
+   o  Specification Document(s): This document
+
+   o  Use Member Value: "HPKE-Receiver"
+   o  Use Description: Key for HPKE receiver role (decapsulation)
```

Comment:
Maybe "Key for HPKE receiver role (encapsulation/decapsulation)"

Reply:
why would the receiver perform encapsulation ?

Reply:
While this text is going to be removed from this PR...

The public part of receiver role key is used for encapsulation/encryption and the private part is used for decapsulation/decryption.

While the public part of sender role key is used to verify the (optional) message authentication, and private part is used to (optionally) authenticate the message.